jdaviz/configs/cubeviz/plugins/viewers.py

```diff
@@ -193,22 +193,34 @@ def __init__(self, *args, **kwargs):
             "flux_viewer_reference_name", "flux-viewer"
         )
 
+    def _is_spatial_subset(self, layer):
+        # spatial subset layers will have the same data-label as the collapsed flux cube
+        ref_data_label = self.layers[0].layer.data.label
+        return (isinstance(getattr(layer.layer, 'subset_state', None), RoiSubsetState)
```

Comment:
Does this work for compound subset? I think this can return things like `glue.core.subset.OrState`.

Also, can/should we consolidate with similar checks like this?

https://github.com/spacetelescope/jdaviz/blob/4e314437814a87fac2781a6693eab843b4ff5605/jdaviz/app.py#L777-L789

Reply:
I haven't seen any problems with it behaving with a compound state... but perhaps if you create one outside the app entirely before attaching it... is that supported?

Reply:
Cubeviz does have `load_regions` but that only lets you load non-compound ones so far.

Reply:
I tried creating compound subset the normal way and didn't see any problem, sorry for the noise.